### PR TITLE
Add docker file

### PR DESCRIPTION
**Pull request by empeje**
Add docker file

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:xenial
+
+
+#install wget and python
+RUN apt-get update && apt-get install -y \
+    wget \
+    python
```
Comment (empeje) [inline Dockerfile]: add this after install stuffs `rm -rf /var/lib/apt/lists/*`

**Review by empeje — COMMENTED** [review]
comments

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:xenial
```
Comment (empeje) [inline Dockerfile]: if possible use `alpine` if not try `debian slim`

**Review by empeje — COMMENTED** [review]
another comments

**Comment by sahilph** [comment]
Changed the docker file in the branch. Lets test it fully before merging

**Comment by empeje** [comment]
no need to close the PR @sahilph .

Open PR intended for discussion and review.

**Comment by sahilph** [comment]
Oh Ok. Sorry @mappuji

**Comment by empeje** [comment]
Seems good, I'll merge it.

**Pull request merged**